src/uproot_browser/plot_view.py

```diff
@@ -38,8 +38,22 @@ def __rich_console__(
     ) -> rich.console.RenderResult:
         width = options.max_width or console.width
         height = options.height or console.height
-        canvas = make_plot(self.item, width, height)
-        rich_canvas = rich.console.Group(*self.decoder.decode(canvas))
+        try:
+            canvas = make_plot(self.item, width, height)
+            rich_canvas = rich.console.Group(*self.decoder.decode(canvas))
+        except Exception:
+            rich_canvas = rich.console.Group(
+                rich.traceback.Traceback(
+                    extra_lines=1,
+                    max_frames=4,  # Can't be less than 4 frames
+                )
+            )
```

Comment:
Does this need the Group wrapper?

Reply:
Yeah, mypy gives `error: Incompatible types in assignment (expression has type "Traceback", variable has type "Group")  [assignment]` otherwise

Reply:
How about declaring `rich_canvas: Union[RenderableType, Segment]` (guessing from https://github.com/Textualize/rich/blob/972dedff546a60642db36831dbb8aaab19077a2c/rich/console.py#L266)? Probably only need one of the two, maybe `RenderableType`?

Reply:
Actually, simpler to just move the yield up

Reply:
See what I just commented (couldn't suggest it since it covered one unchanged line)

Reply:
And then now you can try removing the Group. :)